README.rst

```diff
@@ -5,52 +5,132 @@ plone.staticresources
 Static JS and CSS resources for Plone.
 
 
-This add on contains all JavaScript and CSS resources from Plone.
-Prior to Plone 5.2, they were located in Products.CMFPlone.
+This add on contains all JavaScript and CSS resources used by Plone.
+
+Prior to Plone 5.2, they were located in ``static/`` directory of
+``Products.CMFPlone`` package.
+
+Due to the specific tooling and workflows used to build frontend resources, and
+also the different maintenance and release needs of ``Products.CMFPlone``
+itself, Plone static resources are now distributed in this package. For a
+complete list of reasons, read the§ `PLIP
+1653<https://github.com/plone/Products.CMFPlone/issues/1653>`_.
 
 .. note::
-  A note on yarn: We require the use of yarn, because yarn supports installing the packages in a custom directory, which npm doesn't.
-  Our package directory is located at: ``src/plone/staticresources/static/components``.
 
+  A note on yarn: We require the use of yarn, because it supports installing
+  packages in a custom named directory, which npm doesn't (even using
+  ``--prefix``, ``node_modules`` is hardcoded). Our package directory is
+  located at: ``src/plone/staticresources/static/components``.
+
+
+Target audience
+---------------
+
+This documentation aims at:
+
+Integrators and Developers
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Who want to customize and/or maintain the frontend of a Plone site for their
+customers:
+
+- upgrade existing versions or install new ``npm`` packages
+- integrate new JavaScript/CSS behavior, eventually wrapped in a ``pattern``
+for optimal integration and reuse within Plone frontend ecosystem
+
+
+Themers
```

Comment:
Do you know if we enforce this practice across our documentation? Do we have a page with preferred vocabulary?

Reply:
It's just not an English word, and I don't know how it would be translated to other languages.

The Plone docs have a limited [dictionary of terms, preferred word choice, and spelling](https://docs.plone.org/about/contributing/word_choice.html), but it can be expanded.